sw/device/silicon_creator/rom_ext/imm_section/README.md

```diff
@@ -0,0 +1,78 @@
+# RomExt Immutable Section Release Guide
+
+## Bump the release version
+
+Edit the `IMM_SECTION_VERSION_{MAJOR,MINOR}"` line in the bazel file.
+`sw/device/silicon_creator/rom_ext/imm_section/defs.bzl`
+
+## Build a release bundle
+
+```shell
+$ bazel build --stamp //sw/device/silicon_creator/rom_ext/imm_section:release_bundle
+```
+
+Building with --stamp flag will stamp the built immutable section, including
+the commit hash and the workspace status.
```

Comment:
add a note here saying that we only want to release an immutable section built from a clean workspace?